beets/autotag/mb.py

```diff
@@ -286,6 +288,16 @@ def album_info(release):
             all_tracks.insert(0, medium['pregap'])
 
         for track in all_tracks:
+
+            if ('title' in track['recording'] and
+                    track['recording']['title'] in SKIPPED_TRACKS):
+                continue
+
+            if ('video' in track['recording'] and
+                    track['recording']['video'] == 'true' and
+                    config['match']['ignore_video_tracks'].get(bool) is True):
```

Comment:
No need for `is True`—calling `.get(bool)` is guaranteed to return a bool.

In fact, it's actually possible to drop that part too—Confit automatically gets the truthiness of a view when it's used as a boolean.

Reply:
Fair enough, I changed it